yarn.lock

```diff
@@ -1422,6 +1422,13 @@
     prop-types "^15.6.1"
     react-lifecycles-compat "^3.0.4"
 
+"@react-native-community/async-storage@^1.12.1":
```

Comment:
@andrzej-onja should the yarn lock be in VC?

Reply:
usually it is but I think we should remove it as we are using our server people form outside of onja will have problems running yarn install.
Lets keep it for now as this is internal project